app/models/library_interactive.rb

```diff
@@ -81,11 +81,17 @@ def export
   end
 
   def generate_export_hash
-    Digest::SHA1.hexdigest(export().to_json)
+    export = export()
+    export.except("authoring_guidance", "description", "name")
```

Comment:
this won't actually remove these properties from export it just returns a new value that has them removed if you use the bang version instead: `except!(...)` that will remove them from the export variable.

Reply:
For reference I tested this in the rails console with:
```
irb(main):001:0> example = {hello: "hi", bye: "bye"}
=> {:hello=>"hi", :bye=>"bye"}
irb(main):003:0> examplej = example.as_json
=> {"hello"=>"hi", "bye"=>"bye"}
irb(main):004:0> examplej.except("hello")
=> {"bye"=>"bye"}
irb(main):005:0> examplej
=> {"hello"=>"hi", "bye"=>"bye"}
irb(main):006:0> examplej.except!("hello")
=> {"bye"=>"bye"}
irb(main):007:0> examplej
=> {"bye"=>"bye"}
```